src/components/EventCard/EventCard.js

```diff
@@ -1,29 +1,23 @@
 import React from "react";
+
 import { Card } from "./EventCard.styled";
-import { MdArrowDropDown } from "react-icons/md";
 
-const EventCard = () => {
+const EventCard = ({ eventItem, handleShowDetails }) => {
+  const { id, title, date, time, description } = eventItem;
   return (
     <li>
-      <Card>
-        {/* <div className="date-time">
-          <p>
-            12-Mar-2021, <span>15:40</span>
-          </p>
-          <MdArrowDropDown size={20} />
-        </div>
-        <h3>Event title</h3> */}
-
+      <Card
+        onClick={() => {
+          handleShowDetails(id);
+        }}
```

Comment:
I think you can  have in this component only onClick={handleShowDetails} and in the parent component leave the prop handleShowDetails={() => handleShowDetails(eventItem.id)}